packages/gatsby-telemetry/src/store.js

```diff
@@ -7,84 +6,42 @@ const {
   existsSync,
   unlinkSync,
 } = require(`fs`)
-const { ensureDirSync } = require(`fs-extra`)
-const Configstore = require(`configstore`)
 
 module.exports = class Store {
-  config = new Configstore(`gatsby`, {}, { globalConfigPath: true })
-
-  constructor() {}
-
-  getConfig(key) {
-    if (key) {
-      return this.config.get(key)
-    }
-    return this.config.all
-  }
-
-  updateConfig(...fields) {
-    this.config.set(...fields)
+  constructor(parentFolder) {
+    this.parentFolder = parentFolder
+    this.bufferFilePath = path.join(this.parentFolder, `events.json`)
   }
 
   appendToBuffer(event) {
-    const bufferPath = this.getBufferFilePath()
     try {
-      appendFileSync(bufferPath, event, `utf8`)
+      appendFileSync(this.bufferFilePath, event, `utf8`)
     } catch (e) {
       //ignore
```

Comment:
also log this event? (since later you also add an TODO to log it)

Reply:
We changed the behaviour with the refactor so this shouldn't be relevant anymore